### DISCUSSION TITLE
About the beta version

### DISCUSSION THREAD

**Discussion by katou0427** [streetwriters/notesnook#4806] [General]
Good evening

I am a Japanese person who is not very knowledgeable about IT.

I continue to use nootsnook because it is very easy to use and the encryption is great.

But there's one problem.

I installed the latest beta version to resolve the sync issue, but it said it had an expiration date.

Does this mean that once the deadline has passed, all my notes will be gone?

Also, what does the beta version mean for this app? (I don't really understand the difference with this product)

**Comment by xa4hf8** [comment]
> **_Does this mean that once the deadline has passed, all my notes will be gone?_**

Your notes will not be lost, you will just have to install a more recent version of the app.

> **_Also, what does the beta version mean for this app?_**

You did not mention the beta version number and OS platform, but generally the beta contains features or fixes that have not been thoroughly tested and approved for mass distribution, so the application could possibly exhibit some unwanted or unexpected behavior.
